src/converters/piral-blazor/package.json

```diff
@@ -57,6 +57,7 @@
     "piral-core": "^0.12.4"
   },
   "peerDependencies": {
-    "piral-core": "0.13.x"
+    "piral-core": "0.13.x",
```

Comment:
I think we can remove:

- everything in `dependencies`
- everything except `piral-core` from `devDependencies`

Not sure, however. Let's give it a try.